plugin/shared/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Dockerile for packaging https://github.com/vmware/docker-volume-vsphere as
+# Docker managed plugin.
```

Comment:
The binary is different.
I will update the docker plugin folder to merge duplicated files.